openshift/templates/pims-db/mssql-deploy.yaml

```diff
@@ -90,6 +90,9 @@ objects:
             - name: mssql-data
               persistentVolumeClaim:
                 claimName: ${APP_NAME}-${COMP_NAME}-${ENV_NAME}${ID}-data
+            - name: backup
```

Comment:
Is this essentially ensuring that when deploying a database it also deploys a backup volume too?  Are persistent volumes within the same pod safe, or can they suffer the same disaster?

Reply:
Have a read through the description of NFS storage on the backup repo:
https://github.com/BCDevOps/backup-container